core/src/main/resources/reference.conf

```diff
@@ -36,6 +36,12 @@ submissionmonitor {
   trackDetailedSubmissionMetrics = true
 }
 
+wdl-parsing {
+  cache-max-size = 2500
```

Comment:
How many WDLs are there in FireCloud? Wondering if we could size our cache to potentially hold everything.

Reply:
currently in prod - 17139 in agora, unknown quantity in dockstor, 10760 distinct references from method configs in a workspace. We could certainly up the cache - I'm totally guessing at a cache size based on how many might be in active use.

Reply:
Is there any way to detect duplicates or is that already unique WDLs?